src/page/HomePage.js

```diff
@@ -4,12 +4,17 @@ import { Link } from 'react-router-dom';
 import { Page } from '../element/Page';
 import { PageBody } from '../element/PageBody';
 import { PageTitle } from '../element/PageTitle';
-import { nextEdition } from '../content/next-edition';
+import { nextEdition } from '../content/editions';
 import { routes } from '../routes';
 
 export class HomePage extends Component {
     render() {
         let upcoming;
+        let warning = {
```

Comment:
aangezien deze niet meer verandert moet het een const zijn
het is misschien handiger om er een component van te maken omdat de styling gedeeld wordt met andere componenten